assets/js/blocks/attribute-filter/edit.js

```diff
@@ -180,6 +181,35 @@ const Edit = ( { attributes, setAttributes, debouncedSpeak } ) => {
 							} )
 						}
 					/>
+					<ToggleControl
+						label={ __(
+							'Filter button',
+							'woo-gutenberg-products-block'
+						) }
+						help={ () => {
+							if ( showFilterButton ) {
+								return __(
+									'Results will only update when the button is pressed.',
+									'woo-gutenberg-products-block'
+								);
+							}
+							return displayStyle === 'list'
+								? __(
+										'Results will update when the options are checked.',
```

Comment:
We could probably use the same string here for both, maybe "Products will update as options are selected".

Reply:
Done in 31ed0281a89691e49dc00d014f9c6030e22c0a92 as well.